### DISCUSSION TITLE
Rule proposal: Prefer styleUrl

### DISCUSSION THREAD

**Discussion by reduckted** [angular-eslint/angular-eslint#1697] [New Rule Proposals]
Angular 17 introduced the `styleUrl: string` property as an alternative to the `styleUrls: string[]` property. A lint rule to enforce the use of `styleUrl` when there's only a single URL would be great.

The `style` property can also be a string instead of a `string[]`, which could be checked by the same rule.

References:
* https://blog.angular.io/introducing-angular-v17-4d7033312e4b#de9c
* https://github.com/angular/angular/pull/51715

**Comment by JamesHenry** [comment]
Please contribute your thoughts directly on the open PR: https://github.com/angular-eslint/angular-eslint/pull/1629
